package.json

```diff
@@ -92,7 +92,6 @@
     "storybook": "^8.5.0",
     "svelte": "^5.12.0",
     "svelte-check": "^4.1.1",
-    "svelte-stripe": "^1.3.0",
```

Comment:
I removed this third-party library and initialized the stripe elements as [recommended by Stripe](https://docs.stripe.com/payments/payment-element#create-a-payment-element).